pkg/okta/okta.go

```diff
@@ -38,13 +41,13 @@ func newJwtVerifier(clientID string, issuer string) *jwtverifier.JwtVerifier {
 	return jwtVerifierSetup.New()
 }
 
-func authorizeMiddleware(next http.Handler, verifier *jwtverifier.JwtVerifier) http.Handler {
+func authorizeMiddleware(logger *zap.Logger, next http.Handler, verifier *jwtverifier.JwtVerifier) http.Handler {
```

Comment:
Once #79 gets merged, we'll want to update this to pull the logger from context so it has the trace ID